csg.js

```diff
@@ -5483,17 +5483,18 @@ for solid CAD anyway.
     // Each side is a line between 2 points
     var CAG = function() {
         this.sides = [];
-        this.isCanonicalized = false;
+        this.isCanonicalized = true;
```

Comment:
That seems like a very significant change , why does the default need to be changed ? It at least impacts a portion of SCAD-API tests and potentially the behaviour of the library